### PR TITLE
FreeBSD support

### PR DESCRIPTION
**Pull request by nullzone**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -102,14 +102,23 @@
...
   # Same package name for both redhat based and debian based
-  package{ $pgbouncer_package_name:
-    ensure  => installed,
-    require => [
-      Class[
-        'postgresql::repo::yum_postgresql_org'],
-        Anchor['pgbouncer::begin']
-      ],
-  }
+
```
Comment (adamcstephens) [inline manifests/init.pp]: This line has a tab on it, which is causing an error in the build.

File: manifests/init.pp [hunk 2]
```diff
@@ -102,14 +102,23 @@
...
-  package{ $pgbouncer_package_name:
-    ensure  => installed,
-    require => [
-      Class[
-        'postgresql::repo::yum_postgresql_org'],
-        Anchor['pgbouncer::begin']
-      ],
-  }
+ 
+  case $::osfamily {
```
Comment (adamcstephens) [inline manifests/init.pp:105]: Please add a default of failure to this case statement, and close it. (Right now its missing the closing brace and thus failing to parse)

**Comment by adamcstephens** [comment]
Thanks for the PR. I'll happily merge once the two issues are resolved.

**Comment by nullzone** [comment]
Done.
Cheers

**Comment by adamcstephens** [comment]
Thanks. We'll get a new release posted soon. I think @cjestel has some more changes he wants in first.

**Comment by cjestel** [comment]
@nullzone can you add another PR that updates the metadata.json file with what versions of FreeBSD this supports. 

I should be able to get a new release cut sometime mid next week.

Thanks for your contribution!

**Comment by nullzone** [comment]
ofc.

Here you go.

**Comment by nullzone** [comment]
I am noticing some uncommited changes in your release which are breaking the module.
Do you want to do a new PS?

**Comment by cjestel** [comment]
I'll take a look at what is there. It may be the temp files from my editor which are in my gitignore file but still got bundled up with the forge module builder program. Those should not break the module though, can you give me the filename(s) in particular that are giving you issues?

**Pull request merged**